flake.nix

```diff
@@ -2,21 +2,40 @@
   description = "soxin: opiniated configs for everyone";
 
   inputs = {
-    nixpkgs.url = "github:NixOS/nixpkgs";
+    deploy-rs.url = github:serokell/deploy-rs;
+    nixpkgs.url = github:NixOS/nixpkgs/release-21.05;
+    nur.url = github:nix-community/NUR;
+    sops-nix.url = github:Mic92/sops-nix;
+    unstable.url = github:NixOS/nixpkgs/nixos-unstable;
+    utils.url = github:gytis-ivaskevicius/flake-utils-plus/v1.1.0;
+
     home-manager = {
       url = "github:nix-community/home-manager";
       inputs.nixpkgs.follows = "nixpkgs";
     };
-    futils.url = "github:numtide/flake-utils";
   };
 
-  outputs = { self, nixpkgs, home-manager, futils } @ inputs:
+  outputs = { nixpkgs, self, utils, ... } @ inputs:
```

Comment:
if you remove an input then the code using the flake will break so I don't think it can easily go un-noticed. Adding them all here makes is frankly annoying 😄

Reply:
Indeed.